tools/loadResources

```diff
@@ -72,12 +72,38 @@ else
 fi
 echo "Determined the json contains resources of type $resType"
 
+
+
 # Loop thru each resource and publish it
 keys=$(jq -r 'to_entries[] | .key' <<< $json)
 checkexitcode $? 'getting keys from the specified json'
 for k in $keys; do
     name=${k#*/}    # strip the org from the beginning of the key
     resourceJson=$(jq -c ".[\"$k\"] | del(.owner) | del(.lastUpdated) | .public = false" <<< $json)
+
+    OLD_HZN_ORG_ID=${k%%/*}
+
+    ## Don't alter service references if it points to an IBM service
+    if [ "${OLD_HZN_ORG_ID}" != "IBM" ] && [ "${OLD_HZN_ORG_ID}" != "${HZN_ORG_ID}"  ]; then
```

Comment:
The various parts of this section should be moved into the relevant case in the case statement below because:
- it is faster to make use of the `resType` we already determined, rather than run another `jq` cmd
- since all of these are resType dependent, the code will be easier to understand

Also, `checkexitcode` only works if it is the next statement after the cmd. You have variable assignments in between, which will always have an exit code of 0.

And why do you put the result of the `jq` cmd into `fixServicesServiceOrgid` and then put that into `resourceJson`. Why not put it straight into `resourceJson` ?